REVIEWERS.md

```diff
@@ -0,0 +1,37 @@
+# Code Reviews
+
+Whether you're contributing or reviewing pull requests, this document is important to you!
+
+- If you're reviewing, it can help you understand what to do.
+- If you're submitting a pull request, it can help you understand what folks are going to do, how to help them, and how to get your pull request merged faster.
+
+Ideas in this document are taken from [Google's Engineering Practices](https://google.github.io/eng-practices/review/). If this is your first time reading the document, start there.
+
+# Creating a PR
+
+- Always branch of `main` for both feature (`feat`) and `bugfix` branches. If you are working together with other developers on a feature and want to each own your branch, create a `feat/{feature}/{GITUB_USERNAME}` branch. Once the `feat/{feature}` has been deleted, we may deleted all nested branches(!).
+
+- Commits must be signed. 
+
+- Fill in the PR template, and make sure to describe your changes well. The developer reviewing your work may be out of the loop on the need of the change, and unfamiliar with the product itself. Linking to a ticket or issue is also acceptable.
+
+- Do not request reviews from too many people. If you are in doubt, let the code owners review. If you specifically know the experts involved, feel free to request up to 3 reviewers.
+
+# For Reviewers
+
+- Be responsive. If you are requested to review, wait at most one business day. Even better to review within the first 4 hours. A review usually is a bit of back and forth, so a 4 hour response time will still lead to a PR taking 2 days to merge.
+
+- For typos and small changes, use the `suggest change` feature.
+
+- Reviewers have an educational function to. Style or small remarks are useful for learnings, but make sure to use `Nit:` as a prefix for the comment, to indicate that the remark is non-essential to the merge of the PR.
+
+In order of importance for code quality:
+
+1. Correctness: It does what it is supposed to do.
+2. Test Coverage: We know that it is correct.
+3. Interfaces and architecture: The signature of a function is more important than it's contents.
+4. Test Brittleness: Good tests verify correctness and public interfaces, and are easy to refactor when the public interface changes
```

Comment:
Maybe out of scope for this doc, but add a note about testing failure cases/ unhappy paths? Success tests are easy to test, but failure testing is just as (if not more!) important and should be included in all test suites

Reply:
Yeah I wanted to keep it relatively short so far, perhaps better on our guide for proptest etc?

Reply:
Makes sense, maybe we could link to https://github.com/ComposableFi/composable/blob/main/docs/CONTRIBUTING.md (or the whole folder) for code expectations when reviewing?